ipatests/test_integration/test_commands.py

```diff
@@ -1561,6 +1561,54 @@ def test_ipa_cacert_manage_prune(self):
 
         assert isrgrootx1_nick in result
 
+    def test_ipa_systemd_journal(self):
+        """
+        This testcase checks that administrative user credentials
+        is not leaked to journald log when ipa commands are run.
+        """
+        tasks.kinit_admin(self.master)
+        tasks.kinit_admin(self.replicas[0])
+        tasks.kinit_admin(self.clients[0])
+        cmds = [
+            ['ipa-adtrust-install', '-a',
+             self.master.config.admin_password, '-U'],
+            ['ipa-replica-manage', '-p',
+             self.master.config.dirman_password, 'list'],
+            ['ipa-csreplica-manage', '-p',
+             self.master.config.dirman_password, 'list'],
+            ['ipa-kra-install', '-p',
+             self.master.config.dirman_password, '-U']
+        ]
+        for host in (self.master, self.replicas[0]):
+            for cmd in cmds:
+                host.run_command(cmd, raiseonerr=False)
+                result = self.master.run_command(
+                    ['journalctl', '-t', f'/usr/sbin/{cmd[0]}',
+                     '-n1', '-o', 'json-pretty'], raiseonerr=False)
+                assert (
+                    self.master.config.admin_password not in result.stdout_text
+                )
+                assert (
+                    self.master.config.dirman_password not in result.stdout_text
+                )
```

Comment:
Please write a function that runs the journalctl command + asserts and reuse everywhere.
```
def check_journal_does_not_contain_secret(host,cmd):
    result = host.run_command(['journalctl', '-t', cmd,
                     '-n1', '-o', 'json-pretty'], raiseonerr=False)
    assert (host.config.admin_password not in result.stdout_text)
    assert (host.config.dirman_password not in result.stdout_text)
```